src/mmw/mmw/settings/base.py

```diff
@@ -138,6 +138,9 @@ def get_env_setting(setting):
 }
 CELERY_TASK_DEFAULT_EXCHANGE = 'tasks'
 CELERY_TASK_DEFAULT_ROUTING_KEY = "task.%s" % STACK_COLOR
+
+CELERY_TASK_TIME_LIMIT = 300
```

Comment:
It looks like in v4, which we're on now, has no default set for time limit:
http://docs.celeryproject.org/en/latest/userguide/configuration.html#task-time-limit

So setting this alone may have an impact, although it looks like the prefix should be CELERY**D** according to 
http://docs.celeryproject.org/en/latest/userguide/configuration.html#new-lowercase-settings  

The setting you have appears to have an effect when tweaked so it's likely backwards-compatible.

Reply:
I'll add the `D` at the end to be safe.

That's weird there's no set default. The error we were seeing on staging spoke of a HardTimeLimit of 300sec being exceeded.

Reply:
It doesn't appear to work for me if I add the `D`...were you able to set it that way? If not, I'm going to open an issue in celery. Could be a mistake in the docs...

Reply:
Nope, but definitely works the way you have it set now, so seems good to leave.